src/test/java/org/folio/circulation/domain/RequestRepresentationTests.java

```diff
@@ -68,17 +75,8 @@ void testExtendedRepresentation() {
   void testStoredRequest() {
     Request request = createMockRequest();
 
-    JsonObject extendedRepresentation
```

Comment:
here's just some copy-paste from the previous test and extended representation should not be used as and input for stored representation, so removed this